### DISCUSSION TITLE
opencpn head predictor is shown but is not changing with wmm enabled while boat is moving

### DISCUSSION THREAD

**Discussion by atchik6056** [bareboat-necessities/lysmarine_gen#433] [Q&A]
raspberry setup with enabled WMM plugin
![Screenshot_2024-07-07_13-27-23](https://github.com/bareboat-necessities/lysmarine_gen/assets/34929784/ecf7b9f7-6de0-4137-bf44-ae56e6ad762c)
head predictor (dotted red with circle) is shown but point to the same direction as above, while COG predictor is showing correctly (solid red with rectangle) when boat is moving
with disabled WMM, head predictor is not shown, but boat points to correct direction corresponding to COG predictor
![Screenshot_2024-07-07_13-29-30](https://github.com/bareboat-necessities/lysmarine_gen/assets/34929784/d8b3589b-9f3e-4049-b5c4-4ec226c13fe8)

the setting is similar to opencpn installed on my laptop, I plug in the same GPS receiver VK-162 USB G-mouse
![Screenshot_2024-07-07_13-36-10](https://github.com/bareboat-necessities/lysmarine_gen/assets/34929784/73898ad2-d174-4657-b11d-17baa70a069e)
opencpn is 5.8.4 on both, but OCPN on the laptop with enabled WMM shows COG predictor only and points boat correctly
There is no electronic compass or fluxgate compass connected, the only VK-162
any comments?

**Comment by mgrouch** [comment]
You can’t have heading display without heading sensor (aka compass or IMU compass)

**Reply by atchik6056** [reply]
i understand it, i surprised to see differences between laptop vs BBN install on RPI with the same settings and OPCN version,
so just double checking
